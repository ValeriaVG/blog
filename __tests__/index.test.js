import React from 'react'
import {shallow} from 'enzyme'
import IndexPage from '../src/pages/index'

describe('Index page', ()=>{
  it('loads without errors',()=>{
    const page = shallow(<IndexPage />)
    expect(page).toMatchSnapshot()
  })
})