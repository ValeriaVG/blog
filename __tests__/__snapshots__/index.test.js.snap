// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page loads without errors 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <IndexPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Hi all
        </h1>,
        <p>
          Welcome to my blog.
        </p>,
        <p>
          That has absolutely nothing.... Except actual unit tests... Yet!
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Hi all",
        },
        "ref": null,
        "rendered": "Hi all",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Welcome to my blog.",
        },
        "ref": null,
        "rendered": "Welcome to my blog.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "That has absolutely nothing.... Except actual unit tests... Yet!",
        },
        "ref": null,
        "rendered": "That has absolutely nothing.... Except actual unit tests... Yet!",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Hi all
          </h1>,
          <p>
            Welcome to my blog.
          </p>,
          <p>
            That has absolutely nothing.... Except actual unit tests... Yet!
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Hi all",
          },
          "ref": null,
          "rendered": "Hi all",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome to my blog.",
          },
          "ref": null,
          "rendered": "Welcome to my blog.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "That has absolutely nothing.... Except actual unit tests... Yet!",
          },
          "ref": null,
          "rendered": "That has absolutely nothing.... Except actual unit tests... Yet!",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
